code/mapentities/TTTLogicAssigned.cs

```diff
@@ -0,0 +1,68 @@
+using Sandbox;
+
+using TTTReborn.Player;
+using TTTReborn.Rounds;
+
+namespace TTTReborn.MapEntities
+{
+    [Library("ttt_logic_assigned", Description = "Used to test the assigned team or role of the activator.")]
+    public partial class TTTLogicAssigned : Entity
+    {
+        [Property("Check Type", "Entity will only return a result for one of the select values.")]
+        public Check CheckType { get; set; } = Check.Role;
```

Comment:
`CheckType` seems a bit of an odd name here?

Reply:
Open for suggestions.

Reply:
Yeah don't have any good suggestions, keep it as is.